js/components/badges/modal.vue

```diff
@@ -11,23 +11,23 @@
         <div class="text-center row">
             <div class="badges col-xs-6 col-xs-offset-3">
                 <button class="btn btn-primary btn-flat btn-block"
-                    v-repeat="badges"
-                    v-on="click: toggle($key)"
-                    v-class="active: selected.indexOf($key) >= 0">
-                    <span class="fa pull-left" v-class="
-                        fa-bookmark: selected.indexOf($key) >= 0,
-                        fa-bookmark-o: selected.indexOf($key) < 0
-                        "></span>
-                    {{ $value }}
+                    v-for="value in badges"
+                    @click="toggle($key)"
+                    :class="{ 'active': selected.indexOf($key) >= 0 }">
+                    <span class="fa pull-left" :class="{
+                        'fa-bookmark': selected.indexOf($key) >= 0,
+                        'fa-bookmark-o': selected.indexOf($key) < 0
+                        }"></span>
+                    {{ value }}
```

Comment:
Maybe even using the `(key, val) in obj` syntax to avoid the implicit/magic too.